webauthn-authenticator-rs/src/lib.rs

```diff
@@ -35,15 +35,17 @@
 //! the appropriate `--features` flag listed inline.
 //!
 //! * [CTAP 2.0, 2.1 and 2.1-PRE protocol implementation][crate::ctap2]
+//! * [Bluetooth][] (with `--features bluetooth`)
 //! * [caBLE][] (with `--features cable`)
-//! * [Mozilla Authenticator][] (with `--features u2fhid`)
 //! * [NFC][] via PC/SC API (with `--features nfc`)
 //! * [SoftPasskey][] (for testing)
 //! * [SoftToken][] (for testing)
 //! * [USB HID][] (with `--features usb`)
+//! * [Mozilla Authenticator][] (with `--features u2fhid`)
```

Comment:
Given how far the support in usb hid and other transports has come, we could consider removing this later.

Reply:
That's something for later that I've previously noted in #214. While our own CTAP2 protocol support is _much_ better, there's usability/UI issues which block it being a true replacement.